lib/trackingParams.js

```diff
@@ -0,0 +1,40 @@
+/** @module trackingParams */
+const {
+    generateDNRRule
+} = require('./utils')
+
+const TRACKING_PARAM_PRIORITY = 20000
```

Comment:
We need to rethink these prioirites slightly, [see the other discussion](https://github.com/duckduckgo/ddg2dnr/pull/49#discussion_r1035658292) (and related Asana task). My suggestion is to go for 40k for this and GPC rules, then 30k for content blocking allowlist rules, but I want to see what you and Sam think first.

Reply:
Done, I made it 40k